ietf/rpc/migrations/0002_populate_rpcrole.py

```diff
@@ -0,0 +1,28 @@
+# Generated by Django 4.2.4 on 2023-08-24 17:31
+
+from django.db import migrations
+
+
+def forward(apps, schema_editor):
+    RpcRole = apps.get_model("rpc", "RpcRole")
+    RpcRole.objects.create(slug="formatting", name="formatting", desc="Formatting an RFC")
+    RpcRole.objects.create(slug="pe", name="primary editor", desc="First editor of an RFC")
```

Comment:
Lets make these slugs longer "primary_editor", etc. - leave a comment about "pe"

Reply:
actually, lets change this to "first_editor" and "name "first editor"